From 2756bacdab41536aaa6664ea77a326ac732788be Mon Sep 17 00:00:00 2001
From: Mike Lothian <mike@fireburn.co.uk>
Date: Mon, 11 May 2020 16:39:14 +0100
Subject: [PATCH] Make IDirect3DShaderValidator9Vtbl extern

This fixes build issues with GCC 10.1 which defaults to -fno-common

/usr/bin/x86_64-pc-linux-gnu-ld: error: d3d9-nine/97cc0d2@@d3d9-nine.dll@sha/shader_validator.c.o: multiple definition of 'IDirect3DShaderValidator9Vtbl'

This fixes https://github.com/iXit/wine-nine-standalone/issues/75
---
 d3d9-nine/shader_validator.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/d3d9-nine/shader_validator.h b/d3d9-nine/shader_validator.h
index 8f52b2e..0d17e1d 100644
--- a/d3d9-nine/shader_validator.h
+++ b/d3d9-nine/shader_validator.h
@@ -17,6 +17,6 @@ typedef struct IDirect3DShaderValidator9Impl
     LONG ref;
 } IDirect3DShaderValidator9Impl;
 
-const void *IDirect3DShaderValidator9Vtbl[6];
+extern const void *IDirect3DShaderValidator9Vtbl[6];
 
 #endif /* __NINE_SHADER_VALIDATOR_H */
